homeassistant/components/control4/__init__.py

```diff
@@ -79,7 +79,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         config[CONF_HOST], director_token_dict[CONF_TOKEN], director_session
     )
     entry_data[CONF_DIRECTOR] = director
-    entry_data[CONF_DIRECTOR_TOKEN_EXPIRATION] = director_token_dict["token_expiration"]
+    entry_data[CONF_DIRECTOR_TOKEN_EXPIRATION] = director_token_dict[
+        CONF_DIRECTOR_TOKEN_EXPIRATION
+    ]
```

Comment:
I couldn't work out where this was actually used.
It seems to be added/updated but never used.
Maybe it should be cleaned up? (as a preliminary PR again...)

Reply:
This is required since the dictionary key to retrieve the number of seconds before expiration changes from "token_expiration" to "validSeconds" in 1.1.0

Reply:
I understand, but from I can see the value `entry_data[CONF_DIRECTOR_TOKEN_EXPIRATION]` is never used in the code.

Reply:
Should be a separate PR... (I know it's a pain but it's a good habit to get into)